src/main/resources/assets/sodium/lang/ru_ru.json

```
@@ -0,0 +1,66 @@
+{
+  "sodium.option_impact.low": "Низкое",
+  "sodium.option_impact.medium": "Среднее",
+  "sodium.option_impact.high": "Высокое",
+  "sodium.option_impact.extreme": "Огромное",
+  "sodium.option_impact.varies": "Разное",
+  "sodium.options.pages.quality": "Качество",
+  "sodium.options.pages.performance": "Производительность",
+  "sodium.options.pages.advanced": "Расширенные",
+  "sodium.options.view_distance.tooltip": "Дальность прорисовки определяет, как далеко будет видна местность. Меньшие значения соответствуют меньшей площади, что улучшает частоту кадров.",
+  "sodium.options.simulation_distance.tooltip": "Дальность симуляции определяет, как далеко местность и сущности будут загружаться и обрабатываться. Меньшие значения могут уменьшить нагрузку на внутренний сервер и увеличить частоту кадров.",
+  "sodium.options.brightness.tooltip": "Управляет яркостью (гаммой) игры.",
+  "sodium.options.gui_scale.tooltip": "Устанавливает коэффициент масштабирования для интерфейса. При выборе \"авто\" всегда будет использоваться наибольший размер.",
+  "sodium.options.fullscreen.tooltip": "Если включено, игра будет открыта во весь экран (если доступно).",
+  "sodium.options.v_sync.tooltip": "Если включена, частота кадров будет синхронизирована с частотой обновления монитора, обеспечивая более плавную игру за счет небольшой задержки ввода. Может понизить производительность слабых систем.",
+  "sodium.options.fps_limit.tooltip": "Ограничивает максимальное количество кадров в секунду. Это поможет уменьшить энергопотребление и нагрузку на систему в обеспечении многозадачности. При включённой вертикальной синхронизации этот параметр будет игнорироваться, если он выше частоты обновления вашего экрана.",
+  "sodium.options.view_bobbing.tooltip": "Если включено, обзор игрока будет покачиваться при движении. Отключение этого параметра может помочь людям, страдающим от укачивания во время игры.",
+  "sodium.options.attack_indicator.tooltip": "Управляет расположением индикатора атаки на экране.",
+  "sodium.options.autosave_indicator.tooltip": "Если включено, при сохранении игрой мира на диск будет показываться индикатор.",
+  "sodium.options.graphics_quality.tooltip": "Качество графики по умолчанию управляет некоторыми наследующими настройками и необходимо для совместимости с модами. Опции ниже со значением \"По умолчанию\" будут использовать этот параметр.",
+  "sodium.options.clouds_quality.tooltip": "Управляет качеством отрисовки облаков в небе.",
+  "sodium.options.weather_quality.tooltip": "Управляет качеством эффектов дождя и снега.",
+  "sodium.options.leaves_quality.name": "Качество листьев",
+  "sodium.options.leaves_quality.tooltip": "Управляет качеством листьев.",
+  "sodium.options.particle_quality.name": "Качество частиц",
+  "sodium.options.particle_quality.tooltip": "Управляет количеством частиц, которые могут присутствовать на экране в любой момент времени.",
```

Improvement from Crowdin: the same meaning, but with better sounding
```suggestion
  "sodium.options.particle_quality.tooltip": "Управляет максимальным количеством частиц, которые могут одновременно присутствовать на экране.",
```